_includes/head.html

```diff
@@ -7,9 +7,20 @@
 ================================================== -->
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  {% unless site.env == "production" %}
-    <meta name="robots" content="noindex, nofollow">
-  {% endunless %}
+
+  {% if page.expiration_date %}
+    {% assign today_date = 'now' | date: '%s' %}
+    {% assign expiration_date = page.expiration_date | date: '%s' %}
+    {% assign expired_content = false %}
+
+    {% if today_date > expiration_date %}
+      {% assign expired_content = true %}
+    {% endif %}
+  {% endif %}
+
+  {% if expired_content == true or site.env != "production" %}
```

Comment:
I see the check for non-prod environments. Was this content being crawled before?

Reply:
Got it, so we **don't** want to index non-production pages (like previews).